### ISSUE TITLE
Anomalous copyright symbol in typing.TypeVar docs

### ISSUE THREAD

**Issue by nmusolino** [python/cpython#92743]
**Documentation**

Currently, the documentation for [`typing.TypeVar`](https://docs.python.org/3/library/typing.html#typing.TypeVar) contains an example that is introduced this way:

> type variables are particularly useful for annotating [classmethods](https://docs.python.org/3/library/functions.html#classmethod) that serve as alternative constructors. In the following example (© [Raymond Hettinger](https://www.youtube.com/watch?v=HTLu2DFOdTg)), the type variable C is bound to the Circle class through the use of a forward reference.

In general, code examples in the documentation are released under a [Zero-Clause BSD License](https://docs.python.org/3/license.html#zero-clause-bsd-license-for-code-in-the-python-release-documentation).

Is the copyright symbol © in this case meant to be a formal claim of copyright over this particular example, or is the symbol being used colloquially to acknowledge the author (@rhettinger)?

If it is being used colloquially, I suggest the phrasing be changed to acknowledge the author of the example without using the copyright symbol (with all its legal connotations), such as:

> In the following example (by Raymond Hettinger) ... 

If the copyright symbol is indeed a formal assertion of copyright, I suggest we replace this example with a snippet that is unencumbered by a private copyright.  For example, the [`typing.Type` docs](https://docs.python.org/3/library/typing.html#typing.Type) contain a similar example:
```python
class User: ...
class BasicUser(User): ...
class ProUser(User): ...
class TeamUser(User): ...

# Accepts User, BasicUser, ProUser, TeamUser, ...
def make_new_user(user_class: Type[User]) -> User:
    # ...
    return user_class()
```

This would ensure code examples in the documentation all share a uniform, well-understood copyright status.

**Comment by nmusolino** [comment]
Actually, I can't find this example in the 3.11 documentation, so presumably it has been removed.  (But I haven't been able to track down when it was apparently removed.)

**Comment by AlexWaygood** [comment]
> Is the copyright symbol © in this case meant to be a formal claim of copyright over this particular example, or is the symbol being used colloquially to acknowledge the author (@rhettinger)?

The latter was my intent, when I was writing this snippet of documentation. I intended merely to give credit to @rhettinger for his excellent example, which I shamelessly stole, due to the fact that I couldn't think of a better one.

Apologies for the confusion; feel free to submit a PR rephrasing it to something clearer :)

**Comment by AlexWaygood** [comment]
> Actually, I can't find this example in the 3.11 documentation, so presumably it has been removed. (But I haven't been able to track down when it was apparently removed.)

It was never _removed_; it was a backport that was altered for the 3.10/3.9 branches. For these branches, the extra example made sense; but it wouldn't have made sense to include the extra example in the 3.11 branch due to the introduction of `typing.Self`.

**Comment by nmusolino** [comment]
> It was never _removed_. The example was added in a backport PR that was altered for the 3.10/3.9 branches. For these branches, the extra example made sense; but it wouldn't have made sense to include the extra example in the 3.11 branch due to the introduction of `typing.Self`.

Ah, thank you for the clarification.  Indeed, this snippet [is present](https://github.com/python/cpython/blob/3.10/Doc/library/typing.rst) in the current 3.10 branch.

I can prepare a PR for the 3.10 branch that simply changes the phrasing in the introductory comment.

**Comment by yourlefthandman** [comment]
Added a simple fix for 3.9 and 3.10 :)
